uc_reports/uc_reports.admin.inc

```diff
@@ -484,8 +497,30 @@ function uc_reports_products_custom() {
     $query->addField('c', 'totalcount');
   }
 
+  if (module_exists('uc_stock')) {
+    $query->leftJoin('uc_product_stock', 's', 'n.nid = s.nid');
+    switch ($args['stock_tracking']) {
+      case 'tracked':
+        $query->condition('s.active', 1);
+        break;
+
+      case 'untracked':
+        $query->condition('s.active', 1, '!=');
+        break;
+
+      default:
```

Comment:
Not needed, but does no harm.

Reply:
Does no harm, but I think the comment improves readability, so I'll leave it in.